test/test-tarantool/call.test.py

```diff
@@ -0,0 +1,217 @@
+"""
+Original Tarantool's test box-py/call.test.py had a problem when output with
+running under Python 2 was not the same as with running under Python 3.
+
+Fixed in commit e97425e46a77a4ac9d102c5f88062cb37e5a8da4
+"test: make box-py/call.test.py compatible with Python 2 and 3".
```

Comment:
Here the test aims to verify test-run, so I would mention 72f8b89332b552cf20b2143e6dae7fe5042a9f79, 110f681ac0a56e3016e9a95eb1db544e353fa812 and d421020eeb1b3c2ec66a49a3be0e8a363dbe8d3d. Problems around porting of the test to Python 3 are less relevant here (at least).

Reply:
added